README.md

```diff
@@ -1,38 +1,43 @@
 # IPTB
-iptb is a program used to manage a cluster of ipfs nodes locally on your
-computer. It allows the creation of up to 1000 or more nodes! Allows for
-various setup options to be selected such as different bootstrapping patterns.
-iptb makes testing networks in ipfs easy!
+`iptb` is a program used to create and manage a cluster of sandboxed IPFS nodes locally on your computer. Spin up 1000s of nodes! It exposes various options, such as different bootstrapping patterns. `iptb` makes testing IPFS networks easy!
 
 ### Usage:
 ```
 NAME:
-	iptb - A new cli application
+	iptb - The IPFS TestBed
```

Comment:
Made this more explicit -- I think having the full command output is useful (consistent /w what they'll see when they run --help themselves)